man/repart.d.xml

```diff
@@ -422,12 +422,22 @@
       <varlistentry>
         <term><varname>CopyBlocks=</varname></term>
 
-        <listitem><para>Takes a path to a regular file, block device node or directory. If specified and the
-        partition is newly created the data from the specified path is written to the newly created
-        partition, on the block level. If a directory is specified the backing block device of the file
-        system the directory is on is determined and the data read directly from that. This option is useful
-        to efficiently replicate existing file systems on the block level on a new partition, for example to
-        build a simple OS installer or OS image builder.</para>
+        <listitem><para>Takes a path to a regular file, block device node or directory, or the special value
+        <literal>auto</literal>. If specified and the partition is newly created the data from the specified
+        path is written to the newly created partition, on the block level. If a directory is specified the
+        backing block device of the file system the directory is on is determined and the data read directly
+        from that. This option is useful to efficiently replicate existing file systems on the block level on
+        a new partition, for example to build a simple OS installer or OS image builder.</para>
+
+        <para>If the special value <literal>auto</literal> is specified the source to copy from is
+        automatically derived from the running system or the image specified with
+        <option>--image=</option>. A partition is searched that matches the configured partition type (as
```

Comment:
The usual: "A partition is searched that …" → "A partition that … is located" (or searched **for**, but I think "located" fits betters).